githooks: enhance Release justification error message

Gurio opened the pull request:
Commits until 19.2 need a release justification. This commit extends
an error message, hinting that release justification might be absent
either in a commit message, or in a PR description (when bors build happens).

Release justification: Not a code change.
Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/40820)
<!-- Reviewable:end -->

Gurio commented:
@asubiotto: could you please review?

Gurio committed:
githooks: enhance Release justification error message

Commits until 19.2 need a release justification. This commit extends
an error message, hinting that release justification might be absent
either in a commit message, or in a PR description (when bors build happens).

Release justification: Not a code change.
Release note: None.

asubiotto commented:
LGTM minus `s/detalize/enhance` in the commit title.

cc @jordanlewis

Gurio commented:
@asubiotto done.
Also CI check has failed two times here: https://teamcity.cockroachdb.com/viewLog.html?buildId=1493062&tab=buildResultsDiv&buildTypeId=Cockroach_UnitTests_Test for me. Is it also a known problem? Because I doubt my change could cause this failure.

Gurio commented:
@asubiotto @jordanlewis: It was indeed some CI issue, and it seem to be fixed now. Could either of you bors?

asubiotto commented:
CI failures are flakiness on master (#40805), definitely not your change. Thanks for the PR!

bors r=asubiotto

asubiotto commented:
Looks like bors didn't pick that up, seems like it's having a bad day.

bors r=asubiotto

Gurio commented:
@asubiotto: could you try again? :)

asubiotto commented:
bors r=asubiotto

Gurio commented:
@asubiotto: thanks!
Maybe you have other tasks that I can do, which are not listed here: https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22 ?
